src/test/scala/org/broadinstitute/dsde/workbench/sam/openam/PostgresAccessPolicyDAOSpec.scala

```diff
@@ -48,25 +50,45 @@ class PostgresAccessPolicyDAOSpec extends FreeSpec with Matchers with BeforeAndA
     val resourceType = ResourceType(resourceTypeName, actionPatterns, roles, ownerRoleName, false)
     val otherResourceType = ResourceType(otherRsourceTypeName, actionPatterns, roles, ownerRoleName, false)
 
+    "upsertResourceTypes" - {
+      "creates resource types in config and is idempotent" in {
+        val config = ConfigFactory.load()
+        val appConfig = AppConfig.readConfig(config)
+        dao.upsertResourceTypes(appConfig.resourceTypes, samRequestContext).unsafeRunSync() should contain theSameElementsAs(appConfig.resourceTypes.map(_.name))
+        dao.upsertResourceTypes(appConfig.resourceTypes, samRequestContext).unsafeRunSync() shouldBe empty
+      }
+
+      "only updates resource types that have changed" in {
+        dao.upsertResourceTypes(Set(resourceType, otherResourceType), samRequestContext).unsafeRunSync() should contain theSameElementsAs(Set(resourceType.name, otherResourceType.name))
+        val updatedResourceType = resourceType.copy(reuseIds = !resourceType.reuseIds)
+        dao.upsertResourceTypes(Set(updatedResourceType), samRequestContext).unsafeRunSync() should contain theSameElementsAs(Set(updatedResourceType.name))
```

Comment:
This should include `otherResourceType` in the set passed to `upsertResourceTypes`